src/videojs.ads.js

```diff
@@ -113,12 +114,14 @@ var
           updateEnded = function() {
             ended = true;
           };
-        player.currentTime(snapshot.currentTime);
```

Comment:
This is in fact caused by an issue with duration reporting in HLS/MSE. In this case, the duration() is returning slightly different inconsistent values throughout the content playback.